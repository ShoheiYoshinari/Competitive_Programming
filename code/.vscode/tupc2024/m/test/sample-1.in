3
5 2 1
12345
5 3 3
12345
10 7 1
3141592653
