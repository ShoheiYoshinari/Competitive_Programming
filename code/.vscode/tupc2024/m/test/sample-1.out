123
1
26
